src/parse.cpp

```diff
@@ -138,15 +135,18 @@ std::string my_plugin::compute_container_id_for_thread(const falcosecurity::tabl
             tr,
             [&](const falcosecurity::table_entry& e)
             {
+                if (!container_id.empty()) {
```

Comment:
I thought that returning `false` would stop the loop, instead it raised an exception :skull: